macOS/DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/NetworkProtectionNavBarPopoverManager.swift

```diff
@@ -213,31 +216,56 @@ final class NetworkProtectionNavBarPopoverManager: NetPPopoverManager {
                 activeSitePublisher: activeSitePublisher,
                 uiActionHandler: uiActionHandler)
 
-            let statusViewModel = NetworkProtectionStatusView.Model(controller: controller,
-                                              onboardingStatusPublisher: onboardingStatusPublisher,
-                                              statusReporter: statusReporter,
-                                              uiActionHandler: uiActionHandler,
-                                              menuItems: { [weak self] in
-
+            let menuItems = { [weak self] () -> [NetworkProtectionStatusView.Model.MenuItem] in
```

Comment:
The diff looks awful here.  To clarify: the call you see that's removed is actually further below.  The real change is I'm creating the menu items on a variable before that call to try and make the code more readable.